tsl/test/expected/dist_hypertable_am.out

```diff
@@ -0,0 +1,136 @@
+SET client_min_messages TO ERROR;
+DROP DATABASE IF EXISTS data_node_1;
+DROP DATABASE IF EXISTS data_node_2;
+DROP DATABASE IF EXISTS data_node_3;
+SET client_min_messages TO NOTICE;
+-- Add data nodes using the TimescaleDB node management API
+SELECT * FROM add_data_node('data_node_1', host => 'localhost',
+                            database => 'data_node_1');
+  node_name  |   host    | port  |  database   | node_created | database_created | extension_created 
+-------------+-----------+-------+-------------+--------------+------------------+-------------------
+ data_node_1 | localhost | 55432 | data_node_1 | t            | t                | t
+(1 row)
+
+SELECT * FROM add_data_node('data_node_2', host => 'localhost',
+                            database => 'data_node_2');
+  node_name  |   host    | port  |  database   | node_created | database_created | extension_created 
+-------------+-----------+-------+-------------+--------------+------------------+-------------------
+ data_node_2 | localhost | 55432 | data_node_2 | t            | t                | t
+(1 row)
+
+SELECT * FROM add_data_node('data_node_3', host => 'localhost',
+                            database => 'data_node_3');
+  node_name  |   host    | port  |  database   | node_created | database_created | extension_created 
+-------------+-----------+-------+-------------+--------------+------------------+-------------------
+ data_node_3 | localhost | 55432 | data_node_3 | t            | t                | t
+(1 row)
+
+GRANT USAGE ON FOREIGN SERVER data_node_1, data_node_2, data_node_3 TO PUBLIC;
+-- Create a new table access method by reusing heap handler
+CREATE ACCESS METHOD test_am TYPE TABLE HANDLER heap_tableam_handler;
+SELECT * FROM test.remote_exec('{ data_node_1, data_node_2, data_node_3 }', $$
+CREATE ACCESS METHOD test_am TYPE TABLE HANDLER heap_tableam_handler;
+$$);
+NOTICE:  [data_node_1]: 
+CREATE ACCESS METHOD test_am TYPE TABLE HANDLER heap_tableam_handler
+NOTICE:  [data_node_2]: 
+CREATE ACCESS METHOD test_am TYPE TABLE HANDLER heap_tableam_handler
+NOTICE:  [data_node_3]: 
+CREATE ACCESS METHOD test_am TYPE TABLE HANDLER heap_tableam_handler
+ remote_exec 
+-------------
+ 
+(1 row)
+
+-- Create distributed hypertable using non-default access method
+CREATE TABLE disttable(time timestamptz NOT NULL, device int) USING test_am;
+SELECT * FROM create_distributed_hypertable('disttable', 'time', 'device', 3);
+ hypertable_id | schema_name | table_name | created 
+---------------+-------------+------------+---------
+             1 | public      | disttable  | t
+(1 row)
+
+-- Make sure that distributed hypertable created on data nodes is
+-- using the correct table access method
+SELECT * FROM test.remote_exec('{ data_node_1, data_node_2, data_node_3 }', $$
+
+SELECT amname AS hypertable_amname
+FROM pg_class cl, pg_am am
+WHERE cl.oid = 'disttable'::regclass
+AND cl.relam = am.oid;
+$$);
```

Comment:
Nit: indent this please, it is not clear that this is a nested statement.

Reply:
I believe currently we have the same styling approach among other calls to `remote_exec`, it would be odd to change only this one. Also adding align here would result in sending a query string to the remote server which would also include it. Generally it is not a big deal, especially for testing.

Reply:
I had to read that code several time just to understand that the other select was part of the `remote_exec` call, but it's just me. It's OK to submit it as it is.